### ISSUE TITLE
MaxTotalConnectionsTest failing

### ISSUE THREAD

**Issue by TomGranot** [AsyncHttpClient/async-http-client#1749]
Stack trace:

```
java.lang.AssertionError: expected [null] but found [java.net.UnknownHostException: DNS name not found [response code 3]]
	at org.asynchttpclient.channel.MaxTotalConnectionTest.testMaxTotalConnections(MaxTotalConnectionTest.java:110)
	at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 35 stack frames
```

Investigating, it relies on actual URLs (https://github.com, https://google.com) which feels like a bad idea.

**Comment by slandelle** [comment]
The real problem is that DNS resolution has serious issues on Travis.

**Comment by TomGranot** [comment]
@slandelle Fails locally too:)

**Comment by TomGranot** [comment]
Closed by #1753.
